storage/container_storage_interface/ephemeral-storage-shared-resource-csi-driver-operator.adoc

```diff
@@ -0,0 +1,43 @@
+:_content-type: ASSEMBLY
+[id="ephemeral-storage-shared-resource-csi-driver-operator"]
+= Shared Resource CSI Driver Operator
+include::_attributes/common-attributes.adoc[]
+:context: ephemeral-storage-shared-resource-csi-driver-operator
+
+toc::[]
+
+
+[role="_abstract"]
+As a cluster administrator, you can use the Shared Resource CSI Driver in {product-title} to provision inline ephemeral volumes that contain the contents of `Secret` or `ConfigMap` objects. This way, pods, other Kubernetes types that expose volume mounts, and {product-title} Builds can securely use the contents of those objects across potentially any namespace in the cluster. To accomplish this, there are currently two types of shared resources: a `SharedSecret` custom resource for `Secret` objects, and a `SharedConfigMap` custom resource for `ConfigMap` objects.
```

Comment:
```suggestion
As a cluster administrator, you can use the Shared Resource CSI Driver in {product-title} to provision inline ephemeral volumes that contain the contents of `Secret` or `ConfigMap` objects. This way, Pods other Kubernetes types that expose volume mounts, and {product-title} Builds can securely use the contents of those objects across potentially any namespace in the cluster. To accomplish this, there are currently two types of shared resources: a `SharedSecret` custom resource for `Secret` objects, and a `SharedConfigMap` custom resource for `ConfigMap` objects.
```

Reply:
Red Hat uses lower-case "pod" for general references to pods.
The phrase, "Pods other Kubernetes types that expose volume mounts," seems like a typo. Do we mean:

```suggestion
As a cluster administrator, you can use the Shared Resource CSI Driver in {product-title} to provision inline ephemeral volumes that contain the contents of `Secret` or `ConfigMap` objects. This way, pods and other Kubernetes types that expose volume mounts, and {product-title} Builds can securely use the contents of those objects across potentially any namespace in the cluster. To accomplish this, there are currently two types of shared resources: a `SharedSecret` custom resource for `Secret` objects, and a `SharedConfigMap` custom resource for `ConfigMap` objects.
```

Reply:
yes that is it @rolfedh looks good